addons/web_editor/models/ir_ui_view.py

```diff
@@ -174,7 +176,12 @@ def _copy_field_terms_translations(self, records_from, name_field_from, record_t
                 record_from[name_field_from],
                 {lang: record_from.with_context(prefetch_langs=True, lang=lang)[name_field_from] for lang in langs if lang != lang_env}
             ))
```

Comment:
I just find these code may not be ideal
If a term is translated in `records_from[0]` but not translated in `records_from[1]`
in the final `extra_translation_dictionary` the term won't be translated.

Maybe we can improve it after this PR

Reply:
Hmm… Yes, I'm not sure how this should be handled in the case of multiple records with different translations for the same term :thinking:

I think we should also check if it's possible to backport this "multi record" fix to `16.0` since the feature is available there too... But yes, this should be a part of a separate PR :+1: